.github/workflows/std.yml

```diff
@@ -95,100 +110,131 @@ jobs:
         with:
           ssh_key: ${{ secrets.SSH_PRIVATE_KEY }}
           user_name: ${{ env.DISCOVERY_USER_NAME }}
           ssh_known_hosts_entry: ${{ env.DISCOVERY_KNOWN_HOSTS_ENTRY }}
-      - name: Configure K8S Cluster Access
-        shell: bash
+      - name: Generate the Diff
         run: |
-          echo "Assuming role '$(aws sts get-caller-identity)' in cluster 'lace-dev-us-east-1'."
-          aws eks update-kubeconfig --name "lace-dev-us-east-1"
-      - name: Show commit
-        shell: bash
-        run: |
-          echo commit: ${{ github.sha }}
-      - uses: divnix/std-action/run@main
+          echo 'export K8S_USER=eks-devs' >.envrc.local
+
+          nix develop .#x86_64-linux.local.envs.main -L --command bash -c '
+            set -euo pipefail
+
+            export AWS_PROFILE="lw"
+            export AWS_REGION="us-east-1"
+
+            echo ==================================================
+
+            aws configure list
+
+            aws sts get-caller-identity
+
+            kubectl cluster-info -n dev-preview
+
+            echo ==================================================
+
+            ### FIXME: change to dev-preview!!!
+
+            nix run -L ".#cardano-services.dev-preprod@us-east-1.plan" | tee k8s-plan.diff
+          '
+      - name: Post Comment on the PR
         env:
-          BRANCH: ${{ github.ref_type == 'branch' && github.head_ref }}
           GH_TOKEN: ${{ github.token }}
-          OWNER_AND_REPO: ${{ github.repository }}
-        with: {ffBuildInstructions: true, remoteStore: "ssh-ng://eu.nixbuild.net"}
+        run: |
+          prNumber=$(cut -d/ -f1 <<<'${{ github.ref_name }}')
+          gh pr comment "$prNumber" --body "$(sed -e '2i\\n```diff' -e '$a```' k8s-plan.diff)"
 
 
-  deploy-to-eu:
-    runs-on: ubuntu-latest
-    needs: [discover, images]
-    name: ${{ matrix.target.jobName }} (us-east-1)
-    env:
-      AWS_REGION: us-east-1
-    permissions:
-      id-token: write
-      contents: read
-      deployments: write
+  deploy-to-us:
+    if: github.event_name == 'push' && github.ref_name == 'dev-preview'
+    needs: [images]
+    concurrency:
+      # Only one deployment at a time per environment, and wait for the previous one to finish:
+      group: deploy-${{ github.ref_name }}
+      cancel-in-progress: false
+    name: Deploy (${{ github.ref_name }})
+    runs-on: ubuntu-22.04
     environment:
-      name: dev-preview
-      url: https://backend.dev-preview.eks.lw.iog.io
-    # Boolean input should be compared with string until https://github.com/actions/runner/issues/2238 resolved
-    if: >
-      fromJSON(needs.discover.outputs.hits).deployments.apply != '{}' && ((github.event_name == 'push' && github.ref == 'refs/heads/master') || inputs.deploy)
-
-    strategy:
-      matrix:
-        target: ${{ fromJSON(needs.discover.outputs.hits).deployments.apply }}
+      name: ${{ github.ref_name }}
+      url: https://${{ github.ref_name }}.dapps.lw.iog.io/
```

Comment:
dapps?

Reply:
Right :facepalm:

Which URL would you suggest? https://backend.dev-preview.eks.lw.iog.io ?

Reply:
Yup, that’s what the current deployments use

Reply:
Please use `https://dev-preview.lw.iog.io/`